scripts/analytics/projectShares.js

```diff
@@ -116,6 +116,13 @@ co(function*() {
   }
   console.log("RESULT: ", (totalViews / _.size(levelSessionViews)).toFixed(3), 'views per completed shareable project.')
 
+  var bestProjects = [];
+  for(var sessionId in levelSessionViews) {
+    bestProjects.push({views: levelSessionViews[sessionId].views, sessionId: sessionId});
+  }
+  bestProjects = _.sortBy(bestProjects, function(project) { return -project.views; });
+  console.log(bestProjects.slice(0, 50));
+
```

Comment:
My git hygiene is so great... I didn't mean to commit that, but since I did, I just finished fixing it up so that it works again and also outputs best projects.